### PR TITLE
Added GnuWin32.Gperf 3.0.1

### PR DESCRIPTION
**Pull request by fujii**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108411)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> ##[warning] Validation concluded that this task needs to go to manual review.  (URL Domain validation)

File: manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GnuWin32.Gperf
+PackageVersion: 3.0.1
+PackageLocale: en-US
+Publisher: GnuWin32
+PackageName: 'GnuWin32: gperf'
```
Comment (stephengillie) [inline manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.locale.en-US.yaml:8]:
```suggestion
PackageName: Gperf
PackageUrl: https://www.gnu.org/software/gperf/gperf.html
```

**Comment by stephengillie** [comment]
Hi @fujii,

Unfortunately, the SourceForge page isn't linked on any of the Gnu pages. Without that linkback, I can't verify that it's controlled by the ISV.

**Comment by fujii** [comment]
> Unfortunately, the SourceForge page isn't linked on any of the Gnu pages. Without that linkback, I can't verify that it's controlled by the ISV. This package doesn't seem directly available from the ISV in a Windows installable format, only as a `.tar.gz` file.

It shouldn't be a problem because it is same with other `GnuWin32` packages and `JernejSimoncic.Wget` package.

File: manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: GnuWin32.Gperf
+PackageVersion: 3.0.1
+Installers:
+- InstallerUrl: https://sourceforge.net/projects/gnuwin32/files/gperf/3.0.1/gperf-3.0.1.exe/download
```
Comment (stephengillie) [inline manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.installer.yaml:7]:
```suggestion
- InstallerUrl: http://ftp.gnu.org/pub/gnu/gperf/gperf-3.0.1.tar.gz
```

Comment (fujii) [reply manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.installer.yaml:7]: It's a source tarball. Not a installer. This suggestion doesn't work.

File: manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  Architecture: x86
+  InstallerType: inno
+  InstallerSha256: E95AD33E7D1C99304D1A4F235F689773F157863A7F39D8F8E7B5A5FA164578D5
```
Comment (ItzLevvie) [inline manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.installer.yaml:10]:
```suggestion
  InstallerSha256: E95AD33E7D1C99304D1A4F235F689773F157863A7F39D8F8E7B5A5FA164578D5
  ProductCode: Gperf-3.0.1_is1
```

File: manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [inline manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.locale.en-US.yaml:8]:
```suggestion
PackageName: 'GnuWin32: Gperf'
```
to match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GnuWin32.Gperf
+PackageVersion: 3.0.1
+PackageLocale: en-US
+Publisher: GnuWin32
+PackageName: 'GnuWin32: Gperf'
```
Comment (stephengillie) [inline manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.locale.en-US.yaml:8]:
```suggestion
PackageName: GnuWin32: Gperf'
PackageUrl: https://gnuwin32.sourceforge.net/packages/gperf.htm
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**